doc/guides/doc-style-guide.md

```diff
@@ -1,4 +1,12 @@
-# Style Guide
+# Style guide
```

Comment:
I would expect this document to include guidance on documentation-related decisions made in previous PRs such as https://github.com/nodejs/node/pull/33692, so I wonder if this document should state that it is a _documentation_ style guide.